example/internal/tmdb_mysql/api/get-actor.sql

```diff
@@ -1,24 +0,0 @@
-select
```

Comment:
add all query files back? if we remove it it won't generate queries orm files

Reply:
I don't mind having them all back.